aii-core/src/main/perl/Shellfe.pm

```diff
@@ -412,45 +426,11 @@ sub run_plugin
     }
 }
 
-# Adds extra options to aii-dhcp configuration. This is the only part
-# of the core AII that has to explicitly look at the profile's values.
-# This part may be removed at a later version, when we have a proper
-# DHCP plug-in or component.
-sub dhcpcfg
-{
-    my ($self, $node, $cmd, $st, $mac) = @_;
-
-    my @dhcpop = (DHCPCFG, MAC, $mac);
-    if ("$cmd" eq CONFIGURE) {
-        my $opts = $st->{configuration}->getElement (DHCPOPTION)->getTree;
-        # check for tftpserver option
-        push(@dhcpop,'--tftpserver',$opts->{tftpserver})
-            if (exists($opts->{tftpserver}));
-        # check for addoptions
-        # addoptions is a single string
-        push(@dhcpop,'--addoptions',$opts->{addoptions})
-            if (exists($opts->{addoptions}));
-        push(@dhcpop, "--$cmd", $node);
-    } elsif ("$cmd" eq REMOVEMETHOD) {
-        push(@dhcpop, "--remove", $node);
-    }
-
-    # Pass debug option
-    if (my $dbg = $self->option('debug') ) {
-        if($dbg =~ m{^(\d+)$}) {
-            push(@dhcpop, '--debug', $1);
-        } else {
-            $self->warn("Invalid debug value $dbg passed (should be integer)");
-        };
-    }
-    return @dhcpop;
-}
-
-# Runs aii-dhcp on the configuration object received as argument. It
+# Runs AII::DHCP with the configuration object received as argument. It
 # uses the MAC of the first card marked with "boot"=true.
 sub dhcp
 {
-    my ($self, $node, $st, $cmd) = @_;
+    my ($self, $node, $st, $cmd, $mgr) = @_;
```

Comment:
can y9ou rename `$mgr` to `$parallel_mgr` or somethig like that? maybe a line of comment that is either undef or an instance of `Parallel`